src/containerapp/azext_containerapp/tests/latest/test_containerapp_scenario.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
```

Comment:
Since  CLI recently migrated `azure-devtools` code to `azure-cli-testsdk` and drop the dependency of `azure-devtools`. Related PR: [#20601](https://github.com/Azure/azure-cli/pull/20601) Therefore, CLI extension also needs to be migrated.
Related PR: #4211
```suggestion
from azure.cli.testsdk.scenario_tests import AllowLargeResponse
```